dwave/system/composites/embedding.py

```diff
@@ -136,6 +137,7 @@ def sample(self, bqm, chain_strength=1.0,
                chain_break_fraction=True,
                embedding_parameters=None,
                return_embedding=False,
+               warnings=None,
```

Comment:
Could we for now default to a module-level constant (when `warnings=None`)?

That would allow for simple external, global control over warnings logging.

Ultimately I would like to implement `dwave.common` or `dwave.logging` module that will provide a general interface for logging stuff like warnings here.

Reply:
If we're doing it for `warnings`, we could do it for embedding as well. That would greatly improve the user experience. Until we address it better with our logging subsystem.

Your thoughts on this?

Reply:
You're imagining something like?
```
dwave.system.composites.embedding.WARNING_LEVEL = dwave.system.warnings.SAVE
dwave.system.composites.embedding.RETURN_EMBEDDING = True
```

If we're going the global route, it might be better to do class variables on the EmbeddingComposite, e.g.
```
EmbeddingComposite.warning_level = dwave.system.warnings.SAVE
EmbeddingComposite.return_embedding = True
```
that way they could also be overwritten safely on instantiated objects.

Reply:
Yes. I like class variables approach.